IPython/html/static/notebook/less/notebook.less

```diff
@@ -5,6 +5,32 @@
         padding-right: 0px;
     }
 }
+/* Sticky header effect. Includes media queries to keep 
+   margin between #site and #header when resizing. 
+ */
+
+.header_notebook {
+    position: fixed;
+    width: 100%;
+}
+
+.site_notebook {
+    margin-top: 9em;
+}
+
+@media screen and (-webkit-min-device-pixel-ratio:0) and (max-width: 636px) {
+      .site_notebook {
+          margin-top: 11em;
+        }
+}
+
+@media (max-width: 633px) {
```

Comment:
Oh, sorry. If you look at the breakpoints in media queries that are used in the notebook, there are many and some of them are not considered standard values (according to device sizes.) Therefore, the responsive design is a bit inconsistent in some places. That's why I couldn't restrict myself to use well known breakpoints.

What do you mean by "compute the margins explicitly from existing values"? And  also "hardcoded values in different units from what they are intended to measure."? Do you refer to `em`?

By the way, I will be testing a new solution that (hopefully) solves all these problems.